src/core.js

```diff
@@ -3,21 +3,34 @@ const teachers = ['Nathan', 'Ed', 'Steve', 'Phil', 'Carlo', 'Lewis', 'Shahzad']
 
 // 1. Using an index on the teachers array, change the value of the
 // fourthTeacher variable below to be the fourth teacher listed in the array
-const fourthTeacher = undefined
+const fourthTeachers = teachers[3];
```

Comment:
are you sure the variable is meant to be called `fourthTeachers` and not `fourthTeacher`?